docs/notebooks/03Define_sections.ipynb

```diff
@@ -64,8 +64,8 @@
    },
    "outputs": [],
    "source": [
-    "# The following command adds the .dts accessor to the xarray Dataset.\n",
-    "from dtscalibration import DtsAccessor   # noqa: E401\n",
+    "# The following line introduces the .dts accessor for xarray datasets\n",
+    "import dtscalibration  # noqa: E401  # noqa: E401\n",
```

Comment:
This is not needed. read_silixa_files will already return an object that has the accessor initialized.